fe/fe-core/src/main/java/org/apache/doris/planner/BrokerScanNode.java

```diff
@@ -522,7 +529,12 @@ private void processFileGroup(
                     rangeDesc.setNumAsString(context.fileGroup.isNumAsString());
                     rangeDesc.setReadJsonByLine(context.fileGroup.isReadJsonByLine());
                 }
-                rangeDesc.setHdfsParams(tHdfsParams);
+                if (rangeDesc.hdfs_params != null && rangeDesc.hdfs_params.getFsName() == null) {
```

Comment:
The origin code calls rangeDesc.setHdfsParams twice in the if and else block. I don't think we need to add a function for the duplicate code, because it is only used in these two places.